Scripts/NuGet/EdFi.Installer.WebApi/Install-EdFiOdsWebApi.psm1

```diff
@@ -390,12 +390,18 @@ function Invoke-TransformWebConfigAppSettings {
             $settings = Get-Content $settingsFile | ConvertFrom-Json | ConvertTo-Hashtable
             $settings.ApiSettings.Mode = $Config.InstallType
             $settings.ApiSettings.Engine = $Config.engine
-            $settings.ApiSettings.Features=$Config.WebApiFeatures.Features
-            $settings.ApiSettings.BearerTokenTimeoutMinutes=$Config.WebApiFeatures.BearerTokenTimeoutMinutes
-            $settings.ApiSettings.ExcludedExtensions=$Config.WebApiFeatures.ExcludedExtensions
-            $EmptyHashTable=@{}
-            $mergedSettings = Merge-Hashtables $settings, $EmptyHashTable
-            New-JsonFile $settingsFile $mergedSettings -Overwrite
+            If ($Config.WebApiFeatures.Features -ne $Null) {
+                foreach ($feature in $Config.WebApiFeatures.Features) {
+                    foreach ($defaultfeature in $settings.ApiSettings.Features) {
+                        If ( $feature.Name -eq $defaultfeature.Name) {
+                            $defaultfeature.IsEnabled =$feature.IsEnabled
+                        }
+                    }
+                }
+            }
+           $settings.ApiSettings.BearerTokenTimeoutMinutes=$Config.WebApiFeatures.BearerTokenTimeoutMinutes
```

Comment:
sorry I didn't catch this before but `BearerTokenTimeoutMinutes` should be up one level higher and not under `ApiSettings` for the api to be able to pick it up see:
https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/1ff8abc70dde9c997ca8acf3c5a22436ebb18969/Application/EdFi.Ods.WebApi/appsettings.json#L15

Reply:
That's OK . I updated it . Thanks